README.md

```diff
@@ -1,24 +1,48 @@
-# README
+[](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# BlogProject
 
-Things you may want to cover:
+> Ruby on Rails based project
 
-* Ruby version
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby
+- RSpec
+- Postgres Sql
 
-* Database creation
+## Getting Started
 
-* Database initialization
+To get a local copy follow these simple example steps.  
 
-* How to run the test suite
+1.- Open the terminal window and clone the repository using this command:  
+`git clone https://github.com/AlejandroNeira98/BlogProject.git` 
 
-* Services (job queues, cache servers, search engines, etc.)
+2.- Change the directory to the school-library directory  
+`cd BlogProject`  
+3.- Install the project's dependencies by running this command:   
+`bundle install`
+4.- To run the application the following command
+`rails s`
```

Comment:
- [ ] Please mention in your steps how to add database credentials, how to create the database, and how to run the migration files. to show how to run your project correctly.